src/applications/accreditation/21a/pages/06-background-information-chapter/index.js

```diff
@@ -1,96 +1,219 @@
 import yesNoPageSchema from '../helpers/yesNoPageSchema';
 import HasAConviction from '../../components/06-background-information-chapter/HasAConviction';
-import convictionDetails from './convictionDetails';
-import courtMartialDetails from './courtMartialDetails';
+import backgroundInformationDetailsPageSchema from '../helpers/backgroundInformationDetailsPageSchema';
+import ExplanationDescription from '../../components/06-background-information-chapter/ExplanationDescription';
+import HasBeenCourtMartialed from '../../components/06-background-information-chapter/HasBeenCourtMartialed';
 import IsUnderCharges from '../../components/06-background-information-chapter/IsUnderCharges';
-import conditionThatAffectsRepresentationDetails from './conditionThatAffectsRepresentationDetails';
-import conditionThatAffectsExaminationDetails from './conditionThatAffectsExaminationDetails';
+import HasBeenTerminatedByVSOrg from '../../components/06-background-information-chapter/HasBeenTerminatedByVSOrg';
+import HasConditionThatAffectsRepresentation from '../../components/06-background-information-chapter/HasConditionThatAffectsRepresentation';
+import ConditionThatAffectsRepresentationDetails from '../../components/06-background-information-chapter/ConditionThatAffectsRepresentationDetails';
 
 /** @type {PageSchema} */
 export default {
   title: 'Background information',
   pages: {
-    yourBackgroundInformation,
+    backgroundInformationIntro,
     hasAConviction: yesNoPageSchema({
       title: 'Conviction',
       path: 'conviction',
       question:
-        'Have you ever been convicted, imprisoned, or sentenced to probation or parole?',
+        'Have you ever been convicted, imprisoned, sentenced to probation or parole?',
       description: HasAConviction,
     }),
-    convictionDetails,
+    convictionDetails: backgroundInformationDetailsPageSchema({
+      title: 'Conviction details',
+      path: 'conviction-details',
+      depends: formData => formData.conviction,
+      question:
+        'Provide the date, a detailed explanation, and the location of the conviction as well as the name and address of the military authority or court involved.',
+      explanationDescription: ExplanationDescription,
+    }),
     hasBeenCourtMartialed: yesNoPageSchema({
       title: 'Court martialed',
       path: 'court-martialed',
       depends: formData => formData?.militaryServiceExperiences?.length,
       question: 'Have you ever been convicted by a military court-martial?',
+      description: HasBeenCourtMartialed,
+    }),
+    courtMartialedDetails: backgroundInformationDetailsPageSchema({
+      title: 'Court martialed details',
+      path: 'court-martialed-details',
+      depends: formData => formData.courtMartialed,
+      question:
+        'Provide the date, a detailed explanation, and the location of the conviction as well as the name and address of the military authority or court involved.',
```

Comment:
Is it worth moving this string to a `const` to DRY it up with L29? No strong preference from me either way since it's minor.